software/rallly.yml

```diff
@@ -1,13 +1,16 @@
 name: Rallly
 website_url: https://rallly.co
-description: Create polls to vote on dates and times (alternative to Doodle).
+description: Open-source Doodle alternative. Poll for availability and schedule group meetings.
```

Comment:
Please keep the (alternative to Doodle) as this is standard throughout most of the descriptions

Reply:
Done 👍